src/components/swap/SwapBox.vue

```diff
@@ -120,6 +120,7 @@ import moment from "moment";
 import ERC20_ABI from "@/constants/abis/erc20.json";
 
 import SelectTokenModal from "@/components/shared/select-token/SelectTokenModal.vue";
+import { RKOVWBTC_TOKEN } from "@/constants/tokens/tokens";
```

Comment:
We definitely need to rename this constant, it's nothing to do with KOVAN. I'd rename it to MULTICHAIN_WBTC_TOKEN